coffee/visitors/utilities.py

```diff
@@ -284,7 +283,9 @@ def default_retval(cls):
     """
 
     def __init__(self, symbols):
+        from coffee.utils import flatten
         self.symbols = symbols
+        self.flatten = flatten
```

Comment:
I'm now importing it directly in the relevant member, that is indeed better.

I didn't know about ``chain``; will probably use it all over the place in a next PR